lib/send_sms.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+require_relative 'order'
+
+class SendSMS
+
+  attr_reader :time
+
+  def initialize(time = Time.now + 3600)
+    @time = time
+  end
+
+  def send_sms
+    client = Twilio::REST::Client.new(ENV["AUTH_SID"], ENV["AUTH_TOKEN"])
+  
+    client.messages.create(
+      from: ENV["TWILIO_MOB"],
+      to: ENV["MY_MOB"],
+      body: "Thank you, your order is being prepared and will be delivered by #{@time}"
+    )
```

Comment:
Some Jazzy stuff going here, really cool!
I saw that you were not sure on the SMS RSpec test for this section, I don't know if this will help but, I wonder if something simple as `respond_to` / `eq` will work in RSpect to check if your code will send SMS?